src/main/java/gregtech/common/covers/CoverFluidVoidingAdvanced.java

```diff
@@ -188,6 +194,12 @@ public void renderCover(CCRenderState renderState, Matrix4 translation, IVertexO
         Textures.FLUID_VOIDING_ADVANCED.renderSided(attachedSide, plateBox, renderState, pipeline, translation);
     }
 
+    @Override
+    public EnumActionResult onSoftMalletClick(EntityPlayer playerIn, EnumHand hand, CuboidRayTraceResult hitResult) {
+        this.isWorkingAllowed = !this.isWorkingAllowed;
+        return EnumActionResult.SUCCESS;
+    }
+
```

Comment:
No need to add this override as `CoverFluidVoiding` which you are inheriting from has function with exactly same content.

Reply:
fixed in [6e79095](https://github.com/GregTechCEu/GregTech/pull/1414/commits/6e79095c6d7f3f4208abc5b69ad49efa68a66ed2)